pkgs/games/doom-ports/doomrunner/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, qtbase
+, qmake
+, makeDesktopItem
+, wrapQtAppsHook
+, fetchFromGitHub
+, fetchurl
+}:
+let
+  desktopItem = makeDesktopItem {
+    name = "DoomRunner";
+    desktopName = "DoomRunner";
+    comment = "Preset-oriented graphical launcher of ZDoom and derivatives";
+    categories = [ "Game" ];
+    icon = "DoomRunner";
+    type = "Application";
+    exec = "DoomRunner";
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "doomrunner";
+  version = "1.7.2";
+
+  src = fetchFromGitHub {
+    owner = "Youda008";
+    repo = "DoomRunner";
+    rev = "v${version}";
+    hash = "sha256-a+f1L7WKdSwMlmXvvIkN+CPvW7Wh0gr4WuXsYKotfPE=";
+  };
+
+  # todo: Change this out if upstream or Flatpak has a new icon file
+  icon = fetchurl {
+    url = "https://raw.githubusercontent.com/mbugni/DoomRunnerFlatpak/main/DoomRunner.128.png";
+    name = "DoomRunner.128.png";
+    hash = "sha256-Lcsduut+yPHMt9UfFGNeNssWzxWiD6DLI37I4WxXoEM=";
+  };
```

Comment:
I'm concerned that LICENSE is not set in this repository.

I found this tip on [Archwiki](https://wiki.archlinux.org/title/Desktop_entries#Converting_icons). How about using ImageMagick to convert the source ico to png like this?

It seems that resizing and converting images is also done by other packages.

Reply:
Good thinking! Added that.